### ISSUE TITLE
Add a CHANGELOG.md file to the repo and start tracking changes

### ISSUE THREAD

**Issue by FejZa** [Microsoft/unitysetup.powershell#151]
This would help to understand what changed actually. I suggest following this guide https://keepachangelog.com/en/1.0.0/

**Comment by jwittner** [comment]
I dig it. The Release PRs have had some notes about all the changes, but they can be hard to find and are more focused on testing the change than communicating it.

So we could keep an Unreleased section accumulating in the 'develop' branch, and then upgrade them when moving that into 'master'.

Would we make everyone who did a PR update the changelog?

**Comment by FejZa** [comment]
Exactly. That's how I am managing it in my repo. Have an [Unreleased] section populate until a new release is made and then assign it the new version.

Everyone doing a PR can add his changes to the [Unreleased]. I understand this adds some additional work to manage things but once people get used to the workflow, it's easy.
